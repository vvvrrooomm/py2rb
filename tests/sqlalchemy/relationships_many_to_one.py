from flask_login import current_user
from sqlalchemy import Column, ForeignKey, Integer, String, Boolean
from sqlalchemy.orm import relationship
from flask_sqlalchemy import SQLAlchemy

db = SQLAlchemy()


class Parent(db.Model):
    __tablename__ = "parent_table"
    id = Column(Integer, primary_key=True)
    child_id = Column(Integer, ForeignKey("child_table.id"))
    child = relationship("Child")


class Child(db.Model):
    __tablename__ = "child_table"
    id = Column(Integer, primary_key=True)