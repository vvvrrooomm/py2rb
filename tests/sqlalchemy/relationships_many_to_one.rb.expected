# frozen_string_literal: true

require 'module'

using PythonPrintEx
using PythonStringCountEx

db = SQLAlchemy()
class Parent < ApplicationRecord
  #id = Column(Integer, primary_key: true)
  #child_id = Column(Integer, ForeignKey("child_table.id"))
  has_many :children #
end
class Child < ApplicationRecord
  #id = Column(Integer, primary_key: true)
end
